packages/odyssey-react/src/components/Status/index.tsx

```diff
@@ -53,18 +53,12 @@ const Status: FunctionComponent<Props> = (props) => {
     variant = "neutral",
   } = props;
 
-  const componentClass = useCx(
-    "ods-status",
-    `is-ods-status-${variant}`,
-    { "is-ods-status-label-hidden": labelHidden }
-  );
-
   return (
-    <div className={componentClass} role="status">
-      <span className="ods-status--label">
+    <div className={styles.status} role="status">
+      <span className={labelHidden ? styles['label-visually-hidden'] : styles.label}>
         {label}
       </span>
-      <span className="ods-status--value">
+      <span className={`${styles.value} ${styles[variant]}`}>
```

Comment:
it is harmless in practice, but we don't have a class for every variant correct? `neutral` for example, the effect would be extra whitespace in this string which is minor but we might consider another approach such as `useCx` if we want to combine classes ourselves, or use the CSS Module `composes` keyword

Reply:
An alternative approach could be you (@jeffbelser-okta) did for [TextInput](https://github.com/okta/odyssey/blob/develop/packages/odyssey-react/src/components/TextInput/index.tsx#L152-L156), in an attempt to be consistent I also did it for [Infobox](https://github.com/okta/odyssey/pull/1017/files#diff-1c68bc6935eddb941aae40f4859475212efb47cb1f3c4cb0e39773cfa5e3cb3bR76-R81). I rather like the approach until we figure out what we want to do in the next iteration, whether its this or something else!